manual_user_install.sh

```diff
@@ -113,7 +115,7 @@ function install_systemd (){
     cp $SCRIPT_DIR/stage-audio/01-speech/files/ovos-audio.service $HOME/.config/systemd/user/
     cp $SCRIPT_DIR/stage-audio/02-voice/files/ovos-dinkum-listener.service $HOME/.config/systemd/user/
     cp $SCRIPT_DIR/stage-phal/01-user/files/ovos-phal.service $HOME/.config/systemd/user/
-    echo $sudoPW |  sudo -S cp $SCRIPT_DIR/stage-phal/02-admin/files/ovos-admin-phal.service /etc/systemd/system/
+    echo $sudoPW |  sudo -S cp $SCRIPT_DIR/stage-phal/02-admin/files/ovos-admin-phal.service /lib/systemd/system/
```

Comment:
I moved this to a `--user` service with `User=root` like buildroot does.  does it not work like that?

https://github.com/OpenVoiceOS/raspbian-ovos/commit/f127f9f5fb51d90a10d7358af6303f82e134d760

Reply:
if you put the admin service here `etc/systemd/user/ovos-admin-phal.service` instead of system, will it start?  I would rather put them all as a user service with that one as `root` user.

Reply:
Got it!!  I will try it and let you know.  Sorry I was dense on this one. Don't worry about insulting me by assuming I know nothing. Especially about github. Now I've figured out that your blue text comments are actually links to examples. As they say north of Boston: "Dawn breaks on Marble Head".  Give me a few days and I will try your configuration as seen in the links. Thanks for your patience. :)